venues/ICML.cc/2017/MLAV/python/admin-init.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+
+import openreview
+import sys, os
+import config
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "/Users/mandler/projects/openreview-scripts/utils"))
```

Comment:
you should use a relative path here - the "os.path.join( ... )" call will turn it into the correct path so that you can call it from anywhere